application/models/ExtensionConfig.php

```diff
@@ -149,9 +149,11 @@ public function isCompatible()
      */
     public static function loadFromFile($file)
     {
-        if (!file_exists($file)) {
-            return null;
-        } else {
+        static $_c = [];
+        if (!array_key_exists($file, $_c)) {
```

Comment:
Why not making
```
if (array_key_exists($file, $_c)) {
  return $_c[$file];
}
```

That would make code a bit more easier to read and analyze, as it would be flatter.